test/test.js

```diff
@@ -759,6 +764,89 @@ describe('serveStatic()', function () {
         .get('/todo/')
         .expect(404, done)
     })
+  });
+
+  (skipSymlinks ? describe.skip : describe)('symlink tests', function () {
```

Comment:
`(skipSymlinks ? describe.skip : describe)`

Clever! Never seen this done before but makes sense :)